### DISCUSSION TITLE
What's the default ecmaVersion of latest eslint?

### DISCUSSION THREAD

**Discussion by syt-honey** [eslint/eslint#15255] [Q&A] [answered]
Hi, I'm looking for the default ecmaVersion of the latest eslint repo, but I didn't find.What should I find?

**my destination:**
I want to upgrade my eslint version to support new feature of ES11(like `?.`,the Optional chaining). 

**my confusion:**
What should I do for my purpose? I'm a novice for in-depth understanding of eslint. Thanks a lot.

**Comment by nzakas** [comment, marked as answer]
The default ecmaVersion is 5. When you set ecmaVersion to latest, you’ll get the most up-to-date setting ESLint has. Currently that is 2022.

**Reply by syt-honey** [reply]
> The default ecmaVersion is 5. When you set ecmaVersion to latest, you’ll get the most up-to-date setting ESLint has. Currently that is 2022.
Thank you very much for your answer, I already know how to solve this problem.

I still want to ask, which directory defines the default version?I found it but didn't find it.

Thanks again😄.

**Reply by nzakas** [reply]
There is no directory that defines this. The value is pulled from espree.latestEcmaVersion.
